Content.Shared/Weapons/Melee/MeleeWeaponComponent.cs

```diff
@@ -63,8 +63,8 @@ public sealed partial class MeleeWeaponComponent : Component
     /// <summary>
     /// Base damage for this weapon. Can be modified via heavy damage or other means.
     /// </summary>
-    [DataField(required:true)]
-    [ViewVariables(VVAccess.ReadWrite)]
+    [DataField(required: true)]
+    [ViewVariables(VVAccess.ReadWrite), AutoNetworkedField]
```

Comment:
this doesnt need to be networked if you have clients predict toggling properly, i think

Reply:
They don't tho. 
I'm struggling as to why though... 
The moment I've made the ItemToggleSystem in the server side, it stopped predicting toggling client side. 
Do I need to make an ItemToggleSystem file client side too?

Reply:
no just have it in shared

Reply:
I wanted a server system to handle changes to things that couldn't be called from Shared, like sharpness and disarm malus. 
Once I made that, the OnUse would only trigger on the server-side UID, until I put up a client side system as well. Now that works properly, but I've encountered another issue - if I have the stun baton try to turn itself off after running out of charge, I raise an event. Problem is that said function is on a server system, so it will only raise said event with the server UID. If that happens, the damage numbers will be different again.
I guess I could end up moving the stun baton function to shared, but that will still make the system rather annoying to port to other systems. Just making it autoNetworkedField will be a good backup to the whole ordeal.